bokehjs/src/coffee/renderer/tile/tile_renderer.coffee

```diff
@@ -85,6 +84,18 @@ class TileRendererView extends PlotWidget
       @_map_data()
       @map_initialized = true
 
+    # brute force way of handling resize or responsive event -------------------------------------------------------------
+    if @height != @map_frame.get('height') or @width != @map_frame.get('width')
```

Comment:
It's also a little unclear that ``@height`` and ``@width`` are undefined on the first render, causing the conditional to evaluate as ``True``. Is there a better way to encapsulate this? Perhaps breaking the body into a function and calling it?